### DISCUSSION TITLE
Modding/Customization of existing PS4/Ps5 OS .To play Games on PCs

### DISCUSSION THREAD

**Discussion by Engineerwaseem89** [InoriRus/Kyty#94] [Ideas]
bro your project is good.But creating a new emulator is really very difficult task.and it requires a lot of effort and time instead can we perform Modding/Customization in existing PS4/Ps5 OS/firmware and install it on any x86/x64 PCs .and make it compatible with our hardwares that could be customized to use with any pcs and we enjoy To play Games on PCs

**Comment by ultimaweapon** [comment]
I think https://github.com/AlexAltea/orbital is what you are looking for.
